package.json

```diff
@@ -35,6 +35,7 @@
     "start": "cross-env NODE_ENV=development webpack-dev-server --hot --inline --config webpack.config.dev.js",
     "test": "npm run test:jest",
     "test:jest": "jest --no-watchman --runInBand",
+    "test:ci": "jest --no-watchman --maxWorkers 2 --coverage",
```

Comment:
We could just include coverage in the `test` or `test:jest` scripts, though in my experience it's been a good idea to have one specific for CI as it may have flags specific to that env